src/locations/locations.service.js

```diff
@@ -33,13 +38,19 @@ class LocationsService {
       options.query_params.state = state
     }
 
-    this.localDB = this.pouchDB('navIntLocationsDB')
-    this.replicationFrom = this.localDB.replicate.from(this.remoteDB, options)
+    if (!this.localDB) {
+      this.localDB = this.pouchDB('navIntLocationsDB')
+    }
 
-    Object.keys(this.onReplicationCompleteCallbacks)
-      .forEach((id) => registerCallback(this.replicationFrom, this.onReplicationCompleteCallbacks[id]))
+    if (!this.replicationFrom) {
+      this.replicationFrom = this.localDB.replicate.from(this.remoteDB, options)
+
+      Object.keys(this.onReplicationCompleteCallbacks)
+        .forEach((id) => registerCallback(this.replicationFrom, this.onReplicationCompleteCallbacks[id]))
+    }
 
     return this.replicationFrom
+      .catch(retry)
```

Comment:
Yeah, I think we'd need to check the error here. There can be some legitimate errors e.g. 500's (server/CORS), 401's (unauthorised; though seems unlikely in this case since we login/refresh the session before replicating locations/products). IIRC, `live: true` uses an exponential backoff + max overall timeout, for e.g. if the user goes offline.